### PR TITLE
Some details about percent encoding in data URIs

### PR DESCRIPTION
**Pull request by Yaffle**
@Fyrd , what is the version used by the original author?
I have tested on Firefox 3.6 and Firefox 4, but I didn't try to find the bug in the https://bugzilla.mozilla.org ...

### CODE REVIEW

File: features-json/datauri.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
   ],
   "bugs":[
     {
-      "description":"Non-base64-encoded SVG data URIs need to be uriencoded to work in IE and Firefox as according to the specification."
+      "description":"Non-base64-encoded SVG data URIs need to be uriencoded to work in IE, Edge and Firefox < 4 as according to the specification. For Edge 18 it is needed to encode '\', '^', '`', '|', and '\x7F'."
```
Comment (Fyrd) [inline features-json/datauri.json:26]:
```suggestion
      "description":"Non-base64-encoded SVG data URIs need to be uriencoded to work in IE, Edge and Firefox < 4 as according to the specification. For Edge 18 it is needed to encode '\\', '^', '`', '|', and '\\x7F'."
```
Needs to escape the slashes to be valid JSON

**Comment by Fyrd** [comment]
@Yaffle Don't know I'm afraid. Doesn't seem too import though so wouldn't worry about it.

**Comment by Fyrd** [comment]
Thanks!

**Comment by Yaffle** [comment]
@Fyrd, thank you!

**Pull request merged**